build.gradle.kts

```diff
@@ -26,10 +27,24 @@ buildscript {
 
 plugins {
   id("com.github.gmazzo.buildconfig") version "2.1.0" apply false
+  alias(libs.plugins.spotless)
 }
 
 apply(plugin = "com.vanniktech.maven.publish.base")
 
+configure<SpotlessExtension> {
+  kotlin {
+    target("**/*.kt")
+    ktlint()
+      .editorConfigOverride(
+        mapOf(
+          "indent_size" to "2",
+          "disabled_rules" to "filename, no-unused-imports, indent, comment-spacing",
```

Comment:
I just disabled these rules as they were causing some issues. They can be enabled in some follow up PRs.